src/app/components/link-custom/link-custom.module.scss

```diff
@@ -0,0 +1,126 @@
+.button {
```

Comment:
I suggest to name it smth like `link-as-btn` because we could have real buttons (buttons as `button`) in our project

Reply:
All the styles there refer to buttons, I suggest leaving the name button, if there are real buttons, we can transfer them to the general styles as they are.